src/tests/api.test.ts

```diff
@@ -1,7 +1,7 @@
 import { startAppServer } from '../server/server'
 import fetch from 'node-fetch'
 
-const TEST_PORT = process.env.PORT || 4998
+const TEST_PORT = 4242
```

Comment:
sometimes, when I run tests, I run it on a different port. I have apps running on 4242 on my local machine, so `process.env.PORT` must remain.

Reply:
I got it. I removed it because it circleCI was failing... 🧐

Reply:
hmmmm.... that sucks.
maybe `process.env.PORT || 4242`?

Reply:
or `process.env.TEST_PORT || 4242`